package.json

```diff
@@ -66,15 +67,15 @@
     "jest-environment-jsdom": "^29.0.0",
     "jest-watch-typeahead": "^2.1.0",
     "js-yaml": "^4.1.0",
-    "nx": "~15.0.0",
-    "prettier": "^2.8.0",
+    "nx": "^15.6.3",
+    "prettier": "^2.8.2",
     "renovate-config-seek": "0.4.0",
     "surge": "^0.23.1",
-    "ts-node": "^10.8.0",
+    "tsx": "^3.12.1",
     "typescript": "^4.5.0"
   },
   "volta": {
-    "node": "16.13.0"
+    "node": "18.13.0"
```

Comment:
If we're bumping this, there are a few `@types/node` packages around too that are still on 16

Reply:
Also there is an `.nvmrc` file at the top level and that still specifies 16